logica.py

```diff
@@ -90,6 +90,14 @@ def GetImportRoot():
     return import_root_env
 
 
+def GetTrinoParameters(a):
+  parametersWithoutValue = ["debug", "disable-compression", "ignore-errors", "insecure", "krb5-disable-remote-service-hostname-canonicalization", "password", "progress"]
```

Comment:
This is what I have currently.  Not fond of the approach but I was checking for the presence of the key to indicate whether it should be added. Are you saying that it would be something like: "password: true" and if true, the parameter is added? 

@Engine("trino",
    user: "sdesai",
    password: None,
    ...);